kubernetes/falco/templates/daemonset.yaml

```diff
@@ -50,12 +49,6 @@ spec:
             - /run/containerd/containerd.sock
             - --cri
             - /run/crio/crio.sock
-            - -K
-            - /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
@FedeDP is the service account token file now automatically discovered?

Reply:
We now have the `k8smeta` plugin: https://github.com/falcosecurity/plugins/tree/master/plugins/k8smeta and its collector: https://github.com/falcosecurity/k8s-metacollector